module-1/src/Components/MovieDeleteForm/MovieDeleteForm.tsx

```diff
@@ -0,0 +1,50 @@
+interface MovieDeleteFormProps {
+  title: string;
+  movie?: Movie | null;
+  closeCallback: (b: boolean) => void;
+}
+export const MovieDeleteForm = ({
+  title,
+  movie,
+  closeCallback,
+}: MovieDeleteFormProps) => {
+  const handleClose = (b: boolean = false) => closeCallback(b);
+
+  const deleteMovie = async () => {
+    let url = `http://localhost:4000/movies/${movie?.id}`;
+    let response = await fetch(url, {
+      method: "DELETE",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+      },
+    });
+    response.ok ? handleClose(true) : handleClose();
```

Comment:
```suggestion
    closeCallback(response.ok);
```
`handleClose` seems to be unnecessary since it doesn't add any logic that needs the scope of the component, you can use `closeCallback` directly.

Reply:
fixed